packages/react-dom/src/__tests__/ReactFunctionComponent-test.js

```diff
@@ -365,7 +386,11 @@ describe('ReactFunctionComponent', () => {
     );
   });
 
+  // TODO: deprecate default props support after we remove defaultProps
+  // from function components
```

Comment:
Didn't we plan to keep them for classes forever?

Reply:
Oh sorry, poor phrasing. I just meant that I need to change this test at some point. Will rephrase!